docs/advanced_topics/adding_reports.rst

```diff
@@ -0,0 +1,126 @@
+Adding reports
+==============
+
+Reports are views with listings of pages matching a specific query. They can also export these listings in spreadsheet format.
+They are found in the `Reports` submenu: by default, the `Locked Pages` report is provided, allowing an overview of locked pages on the site.
+
+It is possible to create your own custom reports in the Wagtail admin. To do this, you will need to subclass
+``wagtail.admin.views.reports.ReportView``, which provides basic listing and spreadsheet export functionality:
+
+.. code-block:: python
+
+    # <project>/views.py
+
+    from wagtail.admin.views.report.ReportView
+
+
+    class CustomReportView(ReportView):
+        pass
+
+
+Defining your report
+~~~~~~~~~~~~~~~~~~~~~
+
+The most important attributes and methods to customise to define your report are:
+
+.. method:: get_queryset(self)
+
+This retrieves the queryset of pages for your report. For example, if you wanted a report on all pages shown in menus:
+
+.. code-block:: python
+
+    # <project>/views.py
+
+    from wagtail.admin.views.report.ReportView
+    from wagtail.core.models import Page
+
+
+    class MenuReportView(ReportView):
+        
+        def get_queryset(self):
+            return Page.objects.in_menu()
+
+.. attribute:: template_name
+
+(string)
+
+The template used to render your report. By default, this is ``"wagtailadmin/reports/base_report.html"``,
+which has a listing based on the explorer views, displaying action buttons, as well as the title,
+time of the last update, status, and specific type of any pages.
+
+.. attribute:: title
+
+(string)
+
+The name of your report, which will be displayed in the header.
+
+.. attribute:: ReportView.header_icon
+
+(string)
+
+The name of the icon, using the standard Wagtail icon names. For example, the locked pages view uses ``"locked"``.
+
+Spreadsheet exports
+-------------------
+
+.. attribute:: list_export
+
+(list)
+
+A list of the fields/attributes for each model which are exported as columns in the spreadsheet view. By default,
+this is identical to the listing fields: the title, time of the last update, status, and specific type of any pages.
+
+.. attribute:: export_heading_overrides
+
+(dictionary)
+
+A dictionary of any fields/attributes in ``list_export`` for which you wish to manually specify a heading for the spreadsheet
+column, and their headings. If unspecified, heading will be taken from the field ``verbose_name`` if applicable, and the
```

Comment:
unspecified, heading -> unspecified, the heading

Reply:
Thanks, will change!